### PR TITLE
destinations page fixes

### PR DESCRIPTION
**Pull request by renau06**
Fixed image cropping on destinations.html and added a no image avaible image if no image exists in google placecs api

### CODE REVIEW

File: byway/src/main/webapp/destinations.html [hunk 1]
```diff
@@ -45,9 +45,9 @@
               placeholder="Enter roadtrip start location"
               required
             />
-          <button class="form-button" id="user-current-address-button" onclick= "getCurrentAddress()">Use current location</button>
+          <button class="form-button" id="user-current-address-button" onclick= "getCurrentAddress()">use current location</button>
```
Comment (j-vent) [inline byway/src/main/webapp/destinations.html:48]: nit: spacing around "="

File: byway/src/main/webapp/destinations.js [hunk 2]
```diff
@@ -155,9 +155,13 @@ function updateLocations(locationData){
...
   let destinationToAdd = document.createElement('div');
   destinationToAdd.className = 'location';
-  
   let destinationPhoto = document.createElement('img');
-  destinationPhoto.src = place.photos[0].getUrl();
+  if(place.photos){
+    destinationPhoto.src = place.photos[0].getUrl();
+  }
+  else{
+      destinationPhoto.src = "/images/nophoto.png";
```
Comment (j-vent) [inline byway/src/main/webapp/destinations.js:163]: Use absolute path reference `images/`

File: byway/src/main/webapp/destinations.html (hunk 1 above)
Comment (renau06) [reply byway/src/main/webapp/destinations.html]: done

File: byway/src/main/webapp/destinations.js (hunk 2 above)
Comment (renau06) [reply byway/src/main/webapp/destinations.js]: done

**Pull request merged**